cli/running/watchdog_test.go

```diff
@@ -157,3 +157,35 @@ func TestWatchdogNotRestartable(t *testing.T) {
 	case <-wdDoneChan:
 	}
 }
+
+func TestWatchdogRereadParams(t *testing.T) {
```

Comment:
I could be wrong, but something similar is tested in other tests. I would like to see an integration test that re-reads the configuration from the file.

Reply:
removed commit